CONFIGURATION.md

```diff
@@ -84,8 +84,10 @@ The other placeholders are specified separately.
   # HTTP proxy server to use to connect to the targets.
   [ proxy_url: <string> ]
 
-  # The preferred IP protocol of the HTTP probe (ip4, ip6).
-  [ preferred_ip_protocol: <string> | default = "ip6" ]
+  # The IP protocol of the HTTP probe (ip4, ip6).
+  [ preferred_ip_protocol: <string> | default = "ip6" *DEPRECATED* ]
+  [ ip_protocol: <string> | default = "ip6" ]
+  [ fallback_ip_protocol: <boolean | default = true> ]
```

Comment:
ip_protocol_fallback would make it clearer that this is a bool, rather than a string.

Reply:
I never liked the name or behavior of this code. With the new code, it's nolonger exactly "preferred", it's just specifying a specific protocol, and having a fallback or not.

I do not want to go in circles on this review.

Reply:
I agree, it's better visually and conceptually.